src/components/BackButton.scss

```diff
@@ -0,0 +1,9 @@
+.button {
```

Comment:
This class name is too generic, I'd make it more specific just in case if doesn't disturb other styles.

I'd also recommed using [TailwindCSS](https://tailwindcss.com/) for styling